lib/services/consumed/autocomplete.js

```diff
@@ -0,0 +1,130 @@
+/* @flow */
+import { CompositeDisposable, Disposable } from "atom";
+
+import type { Store } from "../../store";
+import type WatchesStore from "../../store/watches";
+import type WatchStore from "../../store/watch";
+
+/**
+ * This acts as a global storage for the consumed service.
+ */
+export class AutocompleteWatchEditor {
+  disposables = new CompositeDisposable();
+  /** The `consumed autocompleteWatchEditor` */
+  addAutocompleteToEditor = (
+    editor: atom$TextEditor,
+    labels: Array<string>
+  ) => {
+    return;
+  };
+
+  /**
+   * This function is called on activation of autocomplete, or if autocomplete is
+   * already active, then it is called when hydrogen activates.
+   *
+   * @param {Store} store - The global Hydrogen store.
+   * @param {Function} watchEditor - The function provided by `autocomplete.watchEditor`.
+   * @returns {Disposable} This is for clean up when autocomplete or hydrogen deactivate.
```

Comment:
`-` between `{Disposable}` and `This` ?
Very small thing tho.

Reply:
No its worth it 🤣